src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -657,6 +672,15 @@ public void registrationStateChanged(RegistrationStateChangeEvent evt)
     }
 
     /**
+     * Returns whether health check debug is enabled.
```

Comment:
If you don't want to copy full description add @see and refer to the property which has the full description instead of copy pasting not very useful text. Also on @return you can just write boolean or "true if enabled" instead of copy paste.